api/FileSystemHandle.json

```
@@ -238,7 +238,7 @@
             }
           },
           "status": {
-            "experimental": true,
+            "experimental": false,
```

I don't think this one clears the experimental hurdle yet.

The [_Choosing an experimental status_ data guideline](https://github.com/mdn/browser-compat-data/blob/main/docs/data-guidelines.md#choosing-an-experimental-status) hasn't been updated to address `"preview"` values yet but, intuitively, I think we should treat previews as if it were behind a flag, not as generally available.

```suggestion
            "experimental": true,
```